class Solution {
    public int smallestIndex(int[] nums) {
        for(int i = 0; i < nums.length; i++){
            int sum = 0;
            String s = "" + nums[i];

            for(int j = 0; j < s.length(); j++){
                sum += s.charAt(j)  - '0';
            }
            if(sum == i){
                return i;
            }
        }
        return -1;
    }
}